app/helpers/test_passages_helper.rb

```diff
@@ -0,0 +1,13 @@
+module TestPassagesHelper
+
+  def test_progress(test_passage)
+    current_question_number = test_passage.test.number_of_questions - test_passage.count_remaining
+
+    "#{current_question_number}/#{test_passage.test.number_of_questions}"
```

Comment:
вот это в принципе можно и в хелпере оставить, чуть изменив
```suggestion
    "#{test_passage.current_question_number}/#{test_passage.test.number_of_questions}"
```